# Problema 1

## 1.1

In [2]:
M = 5.9e27      # g
R = 6.4e8       # cm
G = 6.67e-8     # dyn*cm^2/g^2
d = 200         # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 0.0006 cm/s**2


## 1.2

In [3]:
d = 1.28e9 # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 3843.0664 cm/s**2


## 1.3.a


In [4]:
M = 1.9e33 # g
d = 200 # cm
R = 1e7  # cm
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 50692000.0000 cm/s**2


## 1.3.b

In [5]:
print(a/979)

51779.36670071501


Sí, esta es igual a 51779 veces la aceleración de la gravedad, y un ser humano sería destrozado y espaghettificado

## 1.4

In [6]:
M = 1.9e41
R = 2.95e13
a = 2*G*M*d/R**3
print(f"a = {a:.4f} cm/s**2")

a = 0.0002 cm/s**2


# 1.5

En un agujero negro supermasivo, porque la fuerza de marea es mucho menor que la que un ser humano experimenta normalmente en la superficie de la Tierra. Esto plantea la cuestión de que, como viajero espacial, podrías encontrarte atrapado por un agujero negro supermasivo a menos que supieras exactamente cuál era su tamaño de antemano. No tendrías ninguna sensación física de haber cruzado el horizonte de sucesos del agujero negro antes de que fuera demasiado tarde.

# Problema 2

In [1]:
import math

c = 2.9979e10    # Velocidad de la Luz (cm/s)
h = 6.6262e-27   # Constante de Planck (erg * s)
m = 9.1095e-28   # Masa del Electrón (g)
e = 4.80325e-10  # Carga del Electrón (esu)
G = 6.6732e-8    # Constante de Gravitación (dyn * cm^2 / g^2)
M = 1.6726e-24   # Masa del Protón (g)

def solve_problem_2():
    # 1. Constante de entropía de agujero negro
    c1 = (c**3) / (2 * h * G)

    # 2. Constante de radiación gravitacional
    c2 = (32 * (G**5)) / (5 * (c**10))

    # 3. Constante de Thomas-Fermi [2]
    c3 = (324 / 175) * (4 / (9 * math.pi))**(2/3)

    # 4. Sección transversal de dispersión de Thompson
    c4 = (8 * math.pi / 3) * ( (e**2) / (m * (c**2)) )**2
    
    # 5. Límite de Stark
    c5 = (1 / (M**5)) * ( (4 * (math.pi**2) * (e**2) * m) / (h**2) )**2

    # 6. Constante de radiación de Bremstrahlung
    c6 = (32 * (math.pi**2) * (e**6)) / (3 * math.sqrt(2 * math.pi * (m**3)) * c)

    # 7. Constante de Fotoionización
    c7 = (32 * (math.pi**2) * (e**6) * (2 * (math.pi**2) * (e**4) * m)) / ((3**1.5) * (h**3))

    print(f"1. Entropía de agujero negro: {c1:.2e}")
    print(f"2. Radiación gravitacional: {c2:.2e}")
    print(f"3. Thomas-Fermi: {c3:.2e}")
    print(f"4. Dispersión de Thompson: {c4:.2e}")
    print(f"5. Límite de Stark: {c5:.2e}")
    print(f"6. Radiación de Bremstrahlung: {c6:.2e}")
    print(f"7. Fotoionización: {c7:.2e}")

solve_problem_2()

1. Entropía de agujero negro: 3.05e+64
2. Radiación gravitacional: 1.44e-140
3. Thomas-Fermi: 5.03e-01
4. Dispersión de Thompson: 6.65e-25
5. Límite de Stark: 2.73e+135
6. Radiación de Bremstrahlung: 6.26e-25
7. Fotoionización: 2.46e-39


# Problema 3

## 3.1

$$F_g = F_c \Rightarrow M = \frac{R v^2}{G}$$

## 3.2

$$v = \frac{2\pi R}{T}\Rightarrow M = \left(\frac{2\pi}{T}\right)^2\cdot\frac{R^3}{G}$$

## 3.3

In [5]:
import math

def Solve33():
    G = 6.67e-11 # SI
    R = 1737 * 1000 # m
    T = 2 * 60 * 60 # s
    
    M = (2*math.pi/T)**2 * R**3/G
    
    return M
print(f"M = {Solve33():.2e} kg")

M = 5.98e+22 kg


Mediciones más precisas, que tienen en cuenta la influencia de la gravedad de la Tierra y una cronología cuidadosa de los períodos orbitales, arrojan en realidad $7.4 \times 10^{22}$ kilogramos.

## 3.4

$$M_{moon} = 5.98 \times 10^{24}$$

$$M_{earth} = 5.98 \times 10^{22}$$

$$\frac{M_{earth}}{M_{moon}} = \frac{1}{100}$$

La relación de masa real es $\frac{1}{80}$